clippy_lints/src/utils/mod.rs

```diff
@@ -602,6 +602,37 @@ pub fn contains_return(expr: &hir::Expr<'_>) -> bool {
     visitor.found
 }
 
+struct FindMacroCalls<'a> {
+    names: Vec<&'a str>,
```

Comment:
```suggestion
struct FindMacroCalls<'a, 'b> {
    names: &'a [&'b str],
```
If we follow my previous suggestion we can change this